src/core/ConfigManager.cpp

```diff
@@ -565,6 +578,7 @@ void ConfigManager::saveConfigFile()
 
 	QDomElement lmms_config = doc.createElement("lmms");
 	lmms_config.setAttribute("version", m_version);
+	lmms_config.setAttribute("configversion", configVersion());
```

Comment:
If you apply the other changes mentioned, this needs to be changed to:
```suggestion
	lmms_config.setAttribute("configversion", m_configVersion);
```